tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -5402,20 +5407,21 @@ public function rememberGroupsThatExistAtTheStartOfTheScenario():void {
 	 *
 	 * @return void
 	 */
-	public function disableOrEnableUser(string $user, string $otherUser, string $action):void {
+	public function disableOrEnableUser(string $user, string $otherUser, string $action): ResponseInterface {
 		$actualUser = $this->getActualUsername($user);
 		$actualPassword = $this->getPasswordForUser($actualUser);
 		$actualOtherUser = $this->getActualUsername($otherUser);
 
 		$fullUrl = $this->getBaseUrl()
 			. "/ocs/v$this->ocsApiVersion.php/cloud/users/$actualOtherUser/$action";
-		$this->response = HttpRequestHelper::put(
+		$response = HttpRequestHelper::put(
 			$fullUrl,
 			$this->getStepLineRef(),
 			$actualUser,
 			$actualPassword
 		);
 		$this->pushToLastStatusCodesArrays();
+		return $response;
```

Comment:
```suggestion
		return HttpRequestHelper::put(
			$fullUrl,
			$this->getStepLineRef(),
			$actualUser,
			$actualPassword
		);
```

Reply:
Also, make sure that the response is stored wherever `disableOrEnableUser` is used in the When step